Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -619,6 +619,12 @@
     <string name="settings_background_priority_card_button">"Open Device Settings"</string>
     <!-- XHED : settings(background priority) - headline on card about the current status and what to do -->
     <string name="settings_background_priority_card_headline">"Disable prioritized background activity"</string>
+    <!-- XTXT: settings(EFGS consent) - status next to switch under title -->
```

Comment:
Only add the strings to the german text files, even if you use placeholder texts. in this file only empty strings should be added, e.g. `<string name="settings_efgs_consent_status_active" />`

Reply:
relevant for all strings added in this PR

Reply:
And you could reference to our Lorem Ipsum string `lorem_ipsum`, that makes it easier to find this placeholders afterwards.

Reply:
still valide!